backend/controllers/attendance/attendanceStatistics.ts

```diff
@@ -386,12 +385,9 @@ export const attendanceStatisticsFactory = (oauthApi, prisonApi, whereaboutsApi)
       return `${data.paid ? 'Yes' : 'No'} - ${pascalToString(data.absentReason).toLowerCase()}`
     }
 
-    const offendersData = suspendedActivites.map((activity) => {
+    const offendersData = suspendedActivities.map((activity) => {
       const attendanceDetails = suspendedAttendances.attendances.find(
-        (attendance) =>
-          activity.bookingId === attendance.bookingId &&
-          moment(activity.startTime).format('YYYY-MM-DD') === attendance.eventDate &&
-          activity.eventId === attendance.eventId
+        (attendance) => activity.eventId === attendance.eventId
```

Comment:
`eventId` is the primary key for an activity anyway, so no need to filter by extra criteria

Reply:
The `EVENT_ID` comes from the `OFFENDER_COURSE_ATTENDANCES` table.  I think the confusion arises since in order to join to that table one needs to:
```
              JOIN OFFENDER_COURSE_ATTENDANCES OCA ON OCA.OFFENDER_BOOK_ID = OPP.OFFENDER_BOOK_ID AND
                    OCA.EVENT_DATE = CS.SCHEDULE_DATE AND
                    OCA.CRS_SCH_ID = CS.CRS_SCH_ID
```
so if we had a `courseScheduleId` instead we would need to use all three fields.

I have confirmed this by calling the `activities-by-date-range` endpoint for a period of a week and checking that each activity has a different event id.